determined-ai/determined
# fix: incorrect job state api inconsistency for namespace-bound workspace jobs

## Ticket
DET-10426


## Description
Fix incorrect API response for jobs created in workspaces manually bound to a Kubernetes namespace (with Determined).



## Test Plan
- Spin up a Kubernetes cluster and create a namespace `test-ns` in that cluster. 
- Launch Determined with an rm connected to that k8s cluser. 
- create a workspace and bind its namespace to `test-ns`
- create an additional workspace and bind it to an auto-created namespace
- launch a Jupyter notebook in both workspaces (using the resource pool corresponding to the rm of the k8s cluster that you just spun up)
- Verify in the webUI that once the workspace shows up as "Running" in the tasks page, it shows up as scheduled in its corresponding resource pool on the Resource Pool Cards page.



## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.




[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
